### PR TITLE
add into field more optional children

### PR DESCRIPTION
**Pull request by mablancoalvarez**
Just the first child received props, the rest of the children  don`t do that.

cc @griselmatosm

### CODE REVIEW

File: src/Field/index.js [hunk 1]
```diff
@@ -3,15 +3,22 @@ import {default as PT} from 'prop-types'
...
       <div className="ola_field-input">
-        {React.cloneElement(children, { id: id, error, disabled })}
+        {React.cloneElement(elements[0], { id: id, error, disabled })}
       </div>
+      {elements.map((item,idx) => {
+        if (idx >= 1) {
+          return item
+        }
+      })}
       {description && <p className={ cx({ 'ola_field-error': error, 'ola_field-description': !error }) }>{description}</p>}
```
Comment (oscarotero) [inline src/Field/index.js:22]: Hi, you can use `shift()` for a more concise way to achieve this:

```js
<div className="ola_field-input">
    {React.cloneElement(elements.shift(), { id: id, error, disabled })}
   {elements}
</div>
```

**Review by oscarotero — CHANGES_REQUESTED** [review]

File: src/Field/index.js (hunk 1 above)
Comment (pedroparra) [reply src/Field/index.js:22]: Yep

**Review by pedroparra — COMMENTED** [review]
Change map for shift.

**Comment by oscarotero** [comment]
LGTM. The only question is why the rest of children aren't inside `ola_field-input` container?

**Comment by mablancoalvarez** [comment]
We did that because that div is the container only for the input (this is the only one that will receive the props from the parent) so we didn't want the rest of the elements (for example if we added a select as in the app) not to be in the same container.

**Comment by oscarotero** [comment]
Ok, in this case, we have to consider how the margins affects to these new elements. At this moment, the `ola_field-input` element has margins (https://github.com/marketgoo/Ola/blob/development/src/Field/style.css#L32) but the `ola_field-description` don't, so these new elements would be closed to them.
A solution could be adding a margin-top to description and error containers.

**Comment by mablancoalvarez** [comment]
you totally right Oscar, we added a margin-top var(--size-2) to description and error containers, i attached a picture so you can see how look like and if that margin is appropriate or if we need to change it.
Captura de pantalla 2020-09-08 a las 18 05 48

**Comment by oscarotero** [comment]
@mablancoalvarez It's perfect now 👍 . Thank you!!

**Review by oscarotero — APPROVED** [review]

**Pull request merged**